src/note/migrations/0004_notepriority.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10 on 2017-01-15 04:19
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def set_initial_priorities(apps, schema_editor):
+    # set NotePriority initial
+    NotePriority = apps.get_model('note', 'NotePriority')
+    NotePriority.objects.create(name="Normal")
+    NotePriority.objects.create(name="Urgent")
```

Comment:
@aaboffill That's normally correct in our context here, but the default should be more explicit as it is not that easy to trace to the migration code to find the value.

And also - what would happen if a user deletes the "Normal" object by accident?